### DISCUSSION TITLE
transaction did not take place

### DISCUSSION THREAD

**Discussion by Kotugogogo** [Chia-Network/chia-blockchain#17804] [Support]
During the transfer on January 08, 2024, 19:18, 2 XCH were withdrawn from the wallet, but the transaction did not take place. Wallet transaction xch1d0c8pnajtjlzemn7edn8fr30fkz3fpzdfstznj7achz753krw7xslc9lyz

**Comment by BrandtH22** [comment]
Hey @Kotugogogo , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the xch address provided, the only transactions to that address occurred on March 29th. Keep in mind that a wallet has multiple xch addresses so those 2xch were likely associated with another address. 

That being said if a transaction occurred from your wallet without your authorization it sounds as though your device or keyset has been compromised. It will be worth reviewing the security best practices as outlined here https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/ . Also, since the chia blockchain is truly and fully decentralized there is unfortunately no method of recovering stolen funds or compromised keysets.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
